### ISSUE TITLE
The entire screen is lagging, I can't use the side bar to the right

### ISSUE THREAD

**Issue by MrMudkip3660** [juliandescottes/piskel#557]
The entire screen lags. I have to save over and over by pressing ctrl+s to save, then reload the page. On top of that, I cannot access the side bar to the right- so I can't export or import anything. Help anyone?  Please?

**Comment by juliandescottes** [comment]
Needs more information to be investigated.

#1 - What is the resolution of the sprite you are editing, how many frames and layers are you using? Or does this happen right away with a blank 32x32 sprite?

#2 - "cannot access the side bar": do you mean it's not visible or nothing happens when you click on a button?

#3 - which browser are you using? If chrome / firefox or IE, which version?

#4 - since you are asking about performance, what are your computer specs/hardware ?

Overall performance will be improved with the next release (0.9.0) but piskel remains focused on editing small sprites.

**Comment by juliandescottes** [comment]
@MrMudkip3660 can provide more information? Otherwise this issue will be closed.

**Comment by MrMudkip3660** [comment]
1. The blank screen happens right away with the start of any sprite.
1. there are 18 frames, and its 32 by 32 pixels
2. Nothing happens when you click the button.
3. I'm using safari, but i can try chrome

**Comment by MrMudkip3660** [comment]
4. my computer is an OS X Yosemite Mac

Thanks for responding!

**Comment by juliandescottes** [comment]
Thanks for the additional info! Would be interested to know if you have similar issues with another browser.

You mention "any sprite" and "18 frames, 32x32" so I'm not sure how to understand that. Maybe you mean the "blank screen" issue happens with any sprite, but the freeze only happens with your 18-frames sprite? If this is correct, can you send me by email the .piskel file export for this sprite?

Regarding the "blank screen" issue, I recently fixed a bug for this for Safari. Can you check if you still get it?

**Comment by MrMudkip3660** [comment]
Sorry my answers were confusing, let me rephrase that:
1) The blank screen happens with any sprite, including mine- my sprite and
any other sprite has the side bar on the right freeze up. (not on Macbook
Pro)
Basically, any sprites I make (including a new one) have the side bar
freeze up.

Another thing to know is that I tested this with both a Mac and a Macbook
Pro- strangely enough, the Macbook Pro is letting me do my work, which i'm
pretty happy about. All the problems i've had on the Mac aren't occurring
here, but It's harder for me to use than on the Mac. Just a little info
with that.

Do you still need the sprite? I'll send it if so. (I'm using a Macbook Pro
right now to do this)

**Comment by juliandescottes** [comment]
Thanks for the additional info. Yes please send the sprite, can't hurt :)

I guess you have the same Mac OS version on both your Mac and your Macbook Pro ? 
Can you check the safari version is also the same ? (should be 10.0 I think)

Do you happen to have safari extensions installed on the Mac that are not installed on the Macbook Pro?
In this case could you try to disable extensions as explained at http://apple.stackexchange.com/a/258809 ?

**Comment by MrMudkip3660** [comment]
I don't have any extra extensions, and I have the same version.
I'll send the sprite in a bit, but it will be sent from the macbook pro,
not mac.


On Thu, Nov 17, 2016 at 12:41 AM, Julian Descottes <notifications@github.com
